dlt/load/load.py

```diff
@@ -556,12 +566,29 @@ def run(self, pool: Optional[Executor]) -> TRunMetrics:
         schema = self.load_storage.normalized_packages.load_schema(load_id)
         logger.info(f"Loaded schema name {schema.name} and version {schema.stored_version}")
 
+        # prepare load package state context
+        load_package_state = self.load_storage.normalized_packages.get_load_package_state(load_id)
```

Comment:
IMO code below should be a part of the `LoadPackageStateInjectableContext` and also incliude
1. load id
2. package storage instance (normalized packages)

Reply:
Alright, I changed it like this, it looks much nicer this way :)